RELEASE_NOTES.md

```diff
@@ -13,6 +13,7 @@
 * Make a distinction between Order and Trade in the protobuf definitions
 * Introduction of new endpoints to retrieve gridpool trades
 * Addition of new definitions and support for trade state filters and streaming
+* Refactor DeliveryDuration to define a timedelta duration attribute instead of defining the class as Enum
```

Comment:
you might want to update the release-notes as only `DeliveryPeriod` was adapted to take in `timedelta`, not `DeliveryDuration`